sulong/projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/debug/LLVMDebugExprParserTest.java

```diff
@@ -0,0 +1,236 @@
+package com.oracle.truffle.llvm.test.debug;
```

Comment:
Much of this class is copied from `LLVMDebugTestBase`. Ideally, we could merge some of the duplicated code.